src/main/java/InputData/newPlaylistInputData.java

```diff
@@ -1,22 +1,23 @@
 package InputData;
+import Entities.RegularUser;
 import Entities.Song;
 import Entities.User;
 
 public class newPlaylistInputData {
     private final int id;
     private final String playlistName;
-    private final User owner;
+    private final RegularUser owner;
```

Comment:
If you want, you could add a comment explaining why the owner is a RegularOwner (instead of User) is a valid assumption to make.

Reply:
Will do